app/actions/speakEasy.js

```diff
@@ -62,9 +65,31 @@ export function postSpeakEasy(params) {
     dispatch(sendPostSpeakEasy());
 
     return (
-      speakEasyAPI.postSpeakEasy(params, token)
-        .then(dispatch(receivePostSpeakEasy()))
-        .then(browserHistory.push("/speakeasies"))
+      speakEasyApi.postSpeakEasy(params, token)
+        .then(() => dispatch(receivePostSpeakEasy()))
+        .then(() => browserHistory.push("/speakeasies"))
+        .catch(reason => console.log(reason))
+    )
+  }
+}
+
+function sendUpdateSpeakEasy() {
+  return { type: SEND_UPDATE_SPEAK_EASY }
+}
+
+function receiveUpdateSpeakEasy() {
+  return { type: RECEIVE_UPDATE_SPEAK_EASY }
+}
+
+export function updateSpeakEasy(id, params) {
+  return (dispatch, getState) => {
+    const { root: { token } } = getState();
+    dispatch(sendUpdateSpeakEasy());
+
+    return (
+      speakEasyApi.updateSpeakEasy(id, params, token)
+        .then(() => dispatch(receiveUpdateSpeakEasy()))
+        .then(() => browserHistory.push("/speakeasies"))
         .catch(reason => console.log(reason))
```

Comment:
I left it i because I don't handle it yet. It should be taken care of in the next pr :D